apps/browser/src/popup/scss/pages.scss

```
@@ -306,7 +306,7 @@ app-fido2 {
       margin-top: 32px;
 
       .subtitle {
-        font-family: Open Sans;
+        font-family: DM Sans;
```

To we need to use `"` here?
```suggestion
        font-family: "DM Sans";
```